faux-mgs/src/main.rs

```diff
@@ -1457,6 +1481,76 @@ async fn monorail_unlock(
         UnlockChallenge::Trivial { timestamp } => {
             UnlockResponse::Trivial { timestamp }
         }
+        UnlockChallenge::EcdsaSha2Nistp256(data) => {
+            // We'll either accept a pubkey path from the command line arguments
+            // (`--key`), or talk to the SSH agent to get a list of keys and
+            // have the user pick one interactively
```

Comment:
I don't love the interactive feature as it seems much different than what we do elsewhere. Could we do something like what humility does for probes and if there's one probe we use it otherwise we specify by number (humility does `usb-1`, we could do `key-0`)?

Reply:
Okay, I changed it to either taking a public key path (ending in `.pub`), or doing a substring match on the list of keys (so you could specify `oxide-support-001-auth`)